meetings/2017/05-18.md

```diff
@@ -19,11 +22,330 @@ Attendees
+      <h1>Hello World</h1>,
+      <MyCarousel images={['./path/foo.jpg']} interval={1000} />
+    ]
+  }
+}
+
+define(MyComponent)
+```
+
+**5.x**
+
+```js
+// MyComponent.jsx
+import { h, Component, define } from 'skate'
+ 
+class MyComponent extends Component {
+  static get is(){ return 'my-cmp' }
+  renderCallback(){
+    return (
+      <div>
+        <h1>Hello World</h1>
+        <my-carousel images={['./path/foo.jpg']} interval={1000} />
+        {/* OR via string refference */}
+        <MyCrousel.is images={['./path/foo.jpg']} interval={1000} />
+      </div>
+    )
+  }
+}
+
+define(MyComponent)
+```
+
+### render without ShadowDom
+
+implemented by: https://github.com/skatejs/skatejs/pull/1141
+
+> this is especially usefull for support of SSR/non WC native browsers/Leaf Components used within Frameworks
+
+**4.x**
+
+- NOT POSSIBLE
+
+**5.x**
+
+```js
+// MyComponent.jsx
+import { h, Component, define } from 'skate'
+ 
+class MyComponent extends Component {
+  static get is(){ return 'my-cmp' }
+  get renderRoot () {
+    return this
+  }
+  renderCallback(){
+    return (
+      <div>
+        <h1>Hello World</h1>
+      </div>
+    )
+  }
+}
+
+define(MyComponent)
+```
+
+so
+
+```html
+<my-cmp></my-cmp>
+```
+
+will render: 
+
+```html
+<my-cmp>
+  <div>
+    <h1>Hello World</h1>
+  </div>
+</my-cmp>
+```
+
+### Properties API => prop.{Name}() replaced by immutable config objects on props object
+
+**4.x**
+
+```js
+// MyComponent.jsx
+import { h, Component, prop } from 'skate'
+ 
+class MyComponent extends Component {
+  static get is(){ return 'my-cmp' }
+  static get props () {
+    return {
+      myArray: prop.array(),
+      myBoolean: prop.boolean()
+    }
+  }
+}
+
+define(MyComponent)
+```
+
+**5.x**
+
+```js
+import { h, Component, props } from 'skate'
+ 
+class MyComponent extends Component {
+  static get is(){ return 'my-cmp' }
+  static get props () {
+    return {
+      myArray: props.array,
+      myBoolean: props.array
+    }
+  }
+}
+
+define(MyComponent)
+```
+
+### Properties API => get/set are gone from prop API
+
+// Comparison TODO
+
+### Properties API => setting/getting props ( triggering re-render )
+
+**4.x**
+
+```js
+// MyComponent.jsx
+import { h, Component, prop, props } from 'skate'
+ 
+class MyComponent extends Component {
+  static get is(){ return 'my-cmp' }
+  static get props () {
+    return {
+      myArray: prop.array(),
+      myBoolean: prop.boolean()
+    }
+  }
+  someNonPublicApiProp = 'Who are you?'
+
+  _changeProps(){
+
+    props(this,{myBoolean: true})
+    // or just directly
+    this.myBoolean = true
+    
+    console.log(props(this)) // { myArray: [], myBoolean: true }
+
+    props( this, {myArray: ['hello']} )
+      // or just directly
+    this.myArray = ['hello']
+
+    console.log(props(this)) // { myArray: ['hello'], myBoolean: true }
+
+    // this will not trigger re-render
+    this.someNonPublicApiProp = 'Im David'
+  }
+}
+
+define(MyComponent)
+```
+
+**5.x**
```

Comment:
I can't spot the breaking change here. Can we simplify this to the most simple form (should probably do this for all examples)?